snforge_std/src/cheatcodes/tx_info.cairo

```diff
@@ -1,83 +1,49 @@
 use starknet::{ContractAddress, testing::cheatcode, contract_address_const};
 use starknet::info::v2::ResourceBounds;
-use snforge_std::cheatcodes::{CheatSpan, CheatTarget, validate_cheat_target_and_span};
+use snforge_std::cheatcodes::CheatSpan;
 use super::super::_cheatcode::handle_cheatcode;
+use super::execution_info::{cheat_execution_info, Operation, ExecutionInfoMock, TxInfoMock};
```

Comment:
Now that `TxInfoMock` has been moved to `execution_info`, I see no reason for this file to exist, its contents could be in `cheatcodes.cairo`

Reply:
Done in #2102